.circleci/config.yml

```diff
@@ -419,18 +419,17 @@ workflows:
           filters:
             tags:
               only: /^v.*/
-      - build-chainlink:
+      - prepublish_npm:
```

Comment:
I think we want to run prepublish_npm on develop, it's only on CI to make sure that we don't break the publishing of our JS package.

Reply:
Just to be clear, I didn't actually change `prepublish_npm` filters, github is just presenting this change a little weirdly. Is there a change you'd like me to make? I'm sure we could sneak it in ;) ...

Reply:
ah, my bad